install.php

```diff
@@ -609,9 +611,9 @@ function insertTestDataIntoMainTables($connection)
                         ["free_keyword" => "CH5 flux measurements", "isCurated" => "0"],
                 ],
                 "Spatial_Temporal_Coverage" => [
-                        ["latitudeMin" => "53.773072687072634", "latitudeMax" => "56.19295930435612", "longitudeMin" => "49.417527009637524", "longitudeMax" => "57.503464509637524", "Description" => "Ein großes Sedimentbecken in Westaustralien, das reich an fossilen Brennstoffen und bedeutenden Erdöl- und Erdgasvorkommen ist.", "dateTimeStart" => "2024-06-03T12:13", "dateTimeEnd" => "2024-06-09T17:13", "timezone" => "+00:00"],
-                        ["latitudeMin" => "7.357546774322249", "latitudeMax" => "8.836749074314008", "longitudeMin" => "-70.8163484389335", "longitudeMax" => "-69.4979890639335", "Description" => "Eine geologisch aktive Region in den Alpen, geprägt durch die Kollision der eurasischen und afrikanischen Kontinentalplatten, die zur Bildung hoher Gebirgsketten führt.", "dateTimeStart" => "2000-07-23T12:22:55", "dateTimeEnd" => "2001-07-23T12:22:55", "timezone" => "+05:00"],
-                        ["latitudeMin" => "26.40875141688829", "latitudeMax" => "56.19295930435612", "longitudeMin" => "14.852995116766325", "longitudeMax" => "18.566374023016326", "Description" => " Ein aktives vulkanisches Gebiet in Island, bekannt für seine regelmäßigen Ausbrüche, Lavafelder und geothermischen Aktivitäten.", "dateTimeStart" => "2024-06-10T12:35", "dateTimeEnd" => "2024-06-16T18:42", "timezone" => "+02:00"]
+                        ["latitudeMin" => "53.773072687072634", "latitudeMax" => "56.19295930435612", "longitudeMin" => "49.417527009637524", "longitudeMax" => "57.503464509637524", "Description" => "Ein großes Sedimentbecken in Westaustralien, das reich an fossilen Brennstoffen und bedeutenden Erdöl- und Erdgasvorkommen ist.", "dateStart" => "2024-06-03", "dateEnd" => "2024-06-03", "TimeStart" => "", "TimeEnd" => "", "timezone" => "+00:00"],
+                        ["latitudeMin" => "7.357546774322249", "latitudeMax" => "8.836749074314008", "longitudeMin" => "-70.8163484389335", "longitudeMax" => "-69.4979890639335", "Description" => "Eine geologisch aktive Region in den Alpen, geprägt durch die Kollision der eurasischen und afrikanischen Kontinentalplatten, die zur Bildung hoher Gebirgsketten führt.", "dateStart" => "2000-07-23", "dateEnd" => "2024-06-03", "TimeStart" => "12:13", "TimeEnd" => "12:22:55", "timezone" => "+05:00"],
```

Comment:
ein leerer String für TimeStart/TimeEnd produziert 00:00:00 in der DB. Ich würde mich über einen Testdatensatz freuen, der für die Zeiten null hat.

Reply:
Betrifft die Testdaten. Also auch @Ali-GeoFZ

Reply:
> ein leerer String für TimeStart/TimeEnd produziert 00:00:00 in der DB. Ich würde mich über einen Testdatensatz freuen, der für die Zeiten null hat.

können wir machen, aber wird auch 00:00:00 produziert, da der Datentyp 'TIME' ist

Reply:
> > ein leerer String für TimeStart/TimeEnd produziert 00:00:00 in der DB. Ich würde mich über einen Testdatensatz freuen, der für die Zeiten null hat.
> 
> können wir machen, aber wird auch 00:00:00 produziert, da der Datentyp 'TIME' ist

hast du das tatsächlich ausprobiert oder ist das nur eine Vermutung? Also laut [MySQL-Doku](https://dev.mysql.com/doc/refman/8.4/en/date-and-time-type-syntax.html) sollte das nicht passieren.